src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -274,19 +297,24 @@ export class HuiCardPicker extends LitElement {
           })}"
         >
           ${!element || element.tagName === "HUI-ERROR-CARD"
-            ? html`
-                ${this.hass!.localize(
+            ? customCard
+              ? customCard.description ||
+                this.hass!.localize(
+                  `ui.panel.lovelace.editor.cardpicker.no_description`
+                )
+              : this.hass!.localize(
                   `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
-                )}
-              `
-            : html`
-                ${element}
-              `}
+                )
+            : element}
```

Comment:
By inverting the first check, you get more readable code (once prettier has formatted it)

```ts
          ${element && element.tagName !== "HUI-ERROR-CARD"
            ? element :
            customCard ?
              customCard.description ||
                this.hass!.localize(
                  `ui.panel.lovelace.editor.cardpicker.no_description`
                )
              : this.hass!.localize(
                  `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
                )
            }
```

Reply:
Nice!